pythermalcomfort/charts/adaptive_chart.py

```diff
@@ -0,0 +1,225 @@
+import numpy as np
+import pandas as pd
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+from pythermalcomfort.charts.theme import (
+    index_mapping_dictionary,
+)
+
+
+def adaptive_chart(df, show_summary=False, si_ip="si"):
+    """Generates a scatter plot with the Adaptive Comfort Model, ASHRAE 55.
+
+    Parameters
+    ----------
+    df : DataFrame
+        A DataFrame containing the input data. Must have the following columns:
+        't_out' (Outdoor temperature [°C]), 'top' (Operative temperature [°C]),
+        'adaptive_acceptability_80%' (Acceptability for 80% of the occupants),
+        'adaptive_acceptability_90%' (Acceptability for 90% of the occupants).
+
+    show_summary : bool, optional
+        If True, a summary of the data is shown. Default is False.
+
+    si_ip : str, optional
+        The unit system to use. It can be 'si' (default) or 'ip'.
+
+    Returns
+    -------
+    fig : graph_objects.Figure
+        Plotly figure object.
+
+    """
+
+    var = "adaptive"
+    var_metadata = index_mapping_dictionary[var]
+    var_name, var_unit, var_range = (
+        var_metadata["name"],
+        var_metadata[si_ip]["unit"],
+        var_metadata[si_ip]["range"],
+    )
+    cat_color, cat_order = (
```

Comment:
I believe these should be functions within the class and we should not be using strings. Let's discuss this during the meeting.